tests/cancun/eip5656_mcopy/test_mcopy_memory_expansion.py

```diff
@@ -0,0 +1,286 @@
+"""
+abstract: Tests [EIP-5656: MCOPY - Memory copying instruction](https://eips.ethereum.org/EIPS/eip-5656)
+
+    Test copy operations of [EIP-5656: MCOPY - Memory copying instruction](https://eips.ethereum.org/EIPS/eip-5656)
```

Comment:
Maybe we should have a unique title docstring here for the docs

Reply:
Done!